pkg/gardenlet/controller/shoot/shoot/reconciler.go

```diff
@@ -589,7 +594,11 @@ func (r *Reconciler) updateShootStatusOperationStart(
 			startRotationSSHKeypair(shoot, &now)
 		}
 		startRotationObservability(shoot, &now)
-		startRotationETCDEncryptionKey(shoot, &now)
+		if k8sLess134 {
+			startRotationETCDEncryptionKey(shoot, false, &now)
+		} else {
+			startRotationETCDEncryptionKey(shoot, true, &now)
+		}
```

Comment:
```suggestion
		startRotationETCDEncryptionKey(shoot, !k8sLess134, &now)
```

Although, I'm wondering whether we shouldn't treat the `OperationRotateCredentialsStart` and `OperationRotateCredentialsStartWithoutWorkersRollout` operations as if triggering etcd key rotation with `rotate-etcd-encryption-key` annotation